app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMPU925X.java

```diff
@@ -0,0 +1,323 @@
+                sensorMPU925X.setAccelRange(Integer.parseInt(spinnerSensorMPU925X2.getSelectedItem().toString()));
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        try {
+            if (sensorMPU925X != null) {
+                sensorMPU925X.setGyroRange(Integer.parseInt(spinnerSensorMPU925X1.getSelectedItem().toString()));
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        return view;
+    }
+
+    private class SensorDataFetch extends AsyncTask<Void, Void, Void> {
+        double[] dataGyro, dataAccel;
+        double dataTemp;
+        long timeElapsed;
+
+        @Override
+        protected Void doInBackground(Void... params) {
+
+            try {
+                dataGyro = sensorMPU925X.getGyroscope();
+                dataAccel = sensorMPU925X.getAcceleration();
+                dataTemp = sensorMPU925X.getTemperature();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+
+            timeElapsed = (System.currentTimeMillis() - startTime) / 1000;
+
+            entriesax.add(new Entry((float) timeElapsed, (float)dataAccel[0]));
+            entriesay.add(new Entry((float) timeElapsed, (float)dataAccel[1]));
+            entriesaz.add(new Entry((float) timeElapsed, (float)dataAccel[2]));
+
+            entriesgx.add(new Entry((float) timeElapsed, (float)dataGyro[0]));
+            entriesgy.add(new Entry((float) timeElapsed, (float)dataGyro[1]));
+            entriesgz.add(new Entry((float) timeElapsed, (float)dataGyro[2]));
+
+            return null;
+        }
+
+        protected void onPostExecute(Void aVoid) {
+            super.onPostExecute(aVoid);
+            tvSensorMPU925Xax.setText(String.valueOf(dataAccel[0]));
+            tvSensorMPU925Xay.setText(String.valueOf(dataAccel[1]));
+            tvSensorMPU925Xaz.setText(String.valueOf(dataAccel[2]));
+            tvSensorMPU925Xgx.setText(String.valueOf(dataGyro[0]));
+            tvSensorMPU925Xgy.setText(String.valueOf(dataGyro[1]));
+            tvSensorMPU925Xgz.setText(String.valueOf(dataGyro[2]));
+            tvSensorMPU925Xtemp.setText(String.valueOf(dataTemp));
+
+            LineDataSet dataset1 = new LineDataSet(entriesax, "Ax");
+            LineDataSet dataSet2 = new LineDataSet(entriesay, "Ay");
+            LineDataSet dataSet3 = new LineDataSet(entriesaz, "Az");
+
+            LineDataSet dataset4 = new LineDataSet(entriesgx, "Gx");
+            LineDataSet dataSet5 = new LineDataSet(entriesgy, "Gy");
+            LineDataSet dataSet6 = new LineDataSet(entriesgz, "Gz");
+
+
+            dataset1.setColor(Color.BLUE);
+            dataSet2.setColor(Color.GREEN);
+            dataSet3.setColor(Color.RED);
+
+            dataset4.setColor(Color.BLUE);
+            dataSet5.setColor(Color.GREEN);
+            dataSet6.setColor(Color.RED);
+
+            dataset1.setDrawCircles(true);
```

Comment:
```drawCircles``` are ```true``` by default, if you want to not draw circle at data point then you set it to ```false```.